web/src/components/Request.tsx

```diff
@@ -719,11 +719,13 @@ export const RequestCancelButton: React.FC = () => {
   return (
     <ButtonGroup variant="outline" size="sm">
       {!request && <Skeleton rounded="full" w="64px" h="32px" />}
-      {request?.status === "PENDING" && request.grant?.status == "PENDING" && (
-        <Button rounded="full" onClick={handleCancel}>
-          Cancel
-        </Button>
-      )}
+      {/* allow to cancel requests if the grant has not been created yet */}
+      {(request?.status === "PENDING" && request.grant?.status == "PENDING") ||
```

Comment:
@meyerjrr the nesting/brackets doesn't look right here, looks like it will only show the cancel button if grant == undefined